docs/DebuggingGuide.md

```diff
@@ -30,7 +30,13 @@ Additional setup guidance can be found in the [Contributing.md](https://github.c
     `git checkout -b informative-branch-name`
 
 ## Debugging
-0. Install the VS SARIF viewer for [VS 2019](https://marketplace.visualstudio.com/items?itemName=WDGIS.MicrosoftSarifViewer) or [VS 2022](https://marketplace.visualstudio.com/items?itemName=WDGIS.MicrosoftSarifViewer2022).
+0. Install the VS SARIF viewer for [VS 2022](https://marketplace.visualstudio.com/items?itemName=WDGIS.MicrosoftSarifViewer2022).
+   
+   Select `Debug` from the `Solution Configurations` menu:
```

Comment:
>Select [](http://example.com/codeflow?start=3&length=6)

Why are we insisting on the debug solution configuration? Users can debug release as well. The guidance as stated here actually builds release in step #2 and then debug here, which seems a bit duplicative. We could just advise the users to `Build Solution`. `Debug` will be the default for them in any case.

Alternately you could drop the running of BuildAndTest.cmd from the root of the enlistment and just advice jumping into the IDE, selecting `Debug` and building.

In general, a debug build does do better when debugging mostly due to the absence of certain code optimizations that cause unexpected behaviors in the IDE (such as not being able to set breakpoints in some places).